debian/control

```diff
@@ -5,7 +5,7 @@ Maintainer: Toradex Packaging Team <debian-pkg-team@toradex.com>
 Uploaders: Abdur Rehman <abdur.rehman@toradex.com>,
            Carlos Henrique Lima Melara <carlos.melara@toradex.com>,
            Leonardo Held <leonardo.held@toradex.com>
-Build-Depends: debhelper-compat (= 13), libgpuperfcnt
+Build-Depends: debhelper-compat (= 13), ca-certificates, cmake, libgpuperfcnt
```

Comment:
`ca-certificates` does not seems to be a Build dependency of `gputop-imx`. I did build it with only cmake and libgpuperfcnt